### ISSUE TITLE
Android Only

### ISSUE THREAD

**Issue by keithsptr** [poiuytrez/SpeechRecognizer#22]
Hi,

Is this solution only for Android or is it supported for iOS and other devices. I am asking this because I am developing an HTML5 app using intel xdk and this allows you to build to different platforms.

Regards,
Keith

**Comment by Createstandard** [comment]
I don't think it works on IOS, I'm looking for a Cordova speech recognition solution on IOS as well and am having problems finding anything.
